workflow/executor/os-specific/signal_windows.go

```diff
@@ -28,14 +29,7 @@ func Setpgid(a *syscall.SysProcAttr) {
 func Wait(process *os.Process) error {
 	stat, err := process.Wait()
 	if stat.ExitCode() != 0 {
-		var errStr string
-		if err != nil {
-			errStr = err.Error()
-		} else {
-			errStr = "<nil>"
```

Comment:
@alexec Was there any motivation to use `<nil>` here for Windows? Could you review this PR?